src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -98,14 +96,10 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Iterables;
 
+import static org.killbill.billing.plugin.adyen.api.mapping.UserDataMappingService.toUserData;
+
 public class AdyenPaymentPluginApi extends PluginPaymentPluginApi<AdyenResponsesRecord, AdyenResponses, AdyenPaymentMethodsRecord, AdyenPaymentMethods> {
 
-    public static final String PROPERTY_CUSTOMER_ID = "customerId";
```

Comment:
Could we actually keep them here? It's helpful to have all of them in the same spot.